website/docs/artist_hosts_maya.md

```diff
@@ -314,10 +314,18 @@ Example setup:
 
 ![Maya - Point Cache Example](assets/maya-pointcache_setup.png)
 
-:::note Publish on farm
-If your studio has Deadline configured, artists could choose to offload potentially long running export of pointache and publish it to the farm.
-Only thing that is necessary is to toggle `Farm` property in created pointcache instance to True.
-:::
+#### Options
+
+- **Frame Start**: which frame to start the export at.
+- **Frame End**: which frame to end the export at.
+- **Handle Start**: additional frames to export at frame start. Ei. frame start - handle start = export start.
+- **Handle Start**: additional frames to export at frame end. Ei. frame end + handle end = export end.
+- **Step**: frequency of sampling the export. For example when dealing with quick movements for motion blur, a step size of less than 1 might be better.
+- **Refresh**: refresh the viewport when exporting the pointcache. For performance is best to leave off, but certain situations can require to refresh the viewport, for example using the Bullet plugin.
+- **Attr**: specific attributes to publish separated by `;`
```

Comment:
There's also `attrPrefix` by the way.

Anyway, the new publisher #4388 exposes both of these nicely in the UI.

Reply:
```suggestion
- **Attr**: specific attributes to publish separated by `;`.
- **AttrPrefix**: Prefix filter for determining which geometric attributes to write out, separated by `;`.
```